app/MacPorts/config.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+# configuration for rsync
+RSYNC = '/usr/bin/rsync'
+RSYNC_SOURCE = "rsync://rsync.macports.org/macports//trunk/dports/PortIndex_darwin_18_i386/PortIndex.json"
```

Comment:
Call this rather `PORTINDEX_SOURCE` or `PORTINDEX_RSYNC_SOURCE` or `PORTINDEX_JSON_RSYNC_SOURCE` or something like that.
Also remove double slash.